### PR TITLE
Non-empty header field in HTTP

### PR DESCRIPTION
**Pull request by fpalombini**
As suggested in #193

### CODE REVIEW

**Comment by emanjon** [comment]
Need to update example as well

**Comment by emanjon** [comment]
Not on expert on HTPP header formats, but are single quotes '' correct? I think I have only seen double quotes "" in HTTP.

File: draft-ietf-core-object-security.md [hunk 1]
```diff
@@ -920,7 +925,7 @@ Mapping and notation here is based on "Simple Form" (Section 5.4.1.1 of {{RFC807
 [HTTP response -- Proxy to HTTP Client]
 
   HTTP/1.1 200 OK
-  Object-Security: [empty]
+  Object-Security: '' (empty string)
   Body: 00 31 d1 fc f6 70 fb 0c 1d d5 ... [binary]
```
Comment (jimsch) [inline draft-ietf-core-object-security.md]: That should be "" not '' - quoted string is defined in the HTTP headers grammar RFC

**Comment by fpalombini** [comment]
should be fixed now

**Comment by emanjon** [comment]
"The value of the payload"

It is called body in HTTP

**Pull request merged**